### DISCUSSION TITLE
SMS and Email Providers

### DISCUSSION THREAD

**Discussion by rsidhaarth** [Blair2004/NexoPOS#390] [Gastro 4.x Ideas & Issues]
Hi Team,

Email Provider: For transactional email, I notice that you have allowed us to use only one service which is Mailgun. But Mailgun never delivers the email to Inbox. So it will be better if you provide these options and make it configurable in the admin panel.

1) SMTP 

If you give option to enter SMTP details, people can use any provider of their choice

2) SendInBlue

One of the best transactional email service provider who delivers the email to inbox 95% and more

SMS Provider: For transactional SMS, It will be better you integrate MSG91 as they are also one of the best providers and used by millions.

Thanks

**Comment by Blair2004** [comment]
Mailgun so far delivers mails to inbox, but I agree to provide other solutions.

**Comment by boudo2017** [comment]
SMTP is recommended ,please
